docs/usage.md

```diff
@@ -434,6 +474,19 @@ CodeChecker cmd diff -b tmux_master_2017_08_28 -n tmux_master_2017_08_29 --new -
 Please find a [Shell Script](script_daily.md) that can be used
 in a Jenkins or any other CI engine to report new bugs.
 
+###Gerrit Integration <a name="gerrit"></a>
+
+The workflow basd on *Alternative 1)* can be used to implement the gerrit integration with CodeChecker.
```

Comment:
```suggestion
The workflow basd on *Alternative a)* can be used to implement the gerrit integration with CodeChecker.
```

Reply:
No, in this case Alternative 1) is correct. I was referring to to section directly above.